mobile/src/screens/Profile/AllowLocation/index.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  ScrollView,
+  Button,
+  Text,
+  VStack,
+  Box,
+  Flex,
+  Switch,
+} from 'native-base';
+import React from 'react';
+
+import { theme } from '@src/styles/theme';
+
+export const AllowLocation = () => {
+  return (
+    <ScrollView>
+      <VStack width={'100%'} height={500} marginTop={180}>
+        <Text
+          fontSize={'2xl'}
+          textAlign={'center'}
+          marginTop={55}
+          bold
+          color={theme.colors.warmGray700}
```

Comment:
Nas próximas vezes que for utilizar cores, releve a recomendação da documentação do Native Base.

Pois, invés de ter:
```tsx
<Text  color={theme.colors.warmGray700} />
 ```
 Podemos utilizar da seguinte forma
 ```tsx
<Text  color="warmGray.700" />
 ```
 Mas está ótimo e não precisa alterar os componentes que já fizemos, fica como uma recomendação para os próximos componentes novos que a gente for construindo.
 
Saiba mais: https://docs.nativebase.io/utility-first